tutorials/prepare_dataset.md

```diff
@@ -142,8 +142,63 @@ The models in Lit-GPT expect datasets for instruction finetuning in the followin
 ```
 (Note that epending on the task, the `"input"` text can be an empty string, as shown above.)
 
+Custom datasets can be prepared by either creating a new `scripts/prepare_dataset.py` script or reading the dataset
+from a CSV file.
 
-The easiest way to prepare a new dataset is to copy and modify one of the existing dataset preparation scripts:
+&nbsp;
+
+### Preparing Custom Datasets From a CSV File
+
+If you have a CSV file containing the following columns
+
+- `instruction`: Column which will describe the task.
+- `input`: A string holding a special input value for the instruction. This applies to some samples, and in others, this is empty (empty string).
+- `output`: The expected response string.
+
+If any of the columns is missing, then the script will fail to create the dataset. 
+
+Before starting to finetune, you need to read, tokenize, and write the data converted from the CSV in a binary format. The simplest way to prepare the dataset is by simply running:
+
+```bash
+python scripts/prepare_csv.py /path/to/the/.csv file
+```
+An example:
+
+```bash
+python scripts/prepare_csv.py /home/user/Downloads/data.csv
+```
```

Comment:
Repeating this is probably unnecessary, it should be clear from the bash snippet above

```suggestion
```